app/views/eventDetails/eventInformation.html

```diff
@@ -288,7 +288,20 @@ <h2 class="page-title border" translate>Event Information</h2>
     <select
       class="form-control"
       ng-model="conference.type"
-      ng-options="type.id as type.name for type in eventTypes | orderBy:'name':false:sortNamesWithNA"
+      ng-options="type.id as type.name for type in ministryPurposes | orderBy:'name':false:sortNamesWithNA"
```

Comment:
If changing the name of `eventTypes` here we should probably also change `type` for `purpose`

Reply:
I think that 'type' is used by the API for ministry purpose. So I didn't want to break anything... We are adding eventType as a new field.

Reply:
@wrandall22 @dr-bizz Are you able to see any reason why my tests are failing?

Reply:
Looking into it